Packs/Grafana/Integrations/Grafana/Grafana.yml

```diff
@@ -0,0 +1,913 @@
+category: Utilities
+commonfields:
+  id: Grafana
+  version: -1
+configuration:
+- defaultvalue: https://www.example.com
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- additionalinfo: max -> 50
+  defaultvalue: '20'
+  display: Maximum number of incidents to fetch
+  hidden: false
+  name: max_fetch
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: 3 days
+  display: First Fetch Time of Alerts
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- display: Dashboard Id to fetch
+  hidden: false
+  name: dashboard_id
+  required: false
+  type: 0
+- display: Panel Id to fetch
+  hidden: false
+  name: panel_id
+  required: false
+  type: 0
+- display: Alert name to fetch
+  hidden: false
+  name: alert_name
+  required: false
+  type: 0
+- display: State to fetch
+  hidden: false
+  name: state
+  options:
+  - all
+  - no_data
+  - paused
+  - alerting
+  - ok
+  - pending
+  required: false
+  type: 15
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: Grafana alerting service
+display: Grafana
+name: Grafana
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: A comma-separated list of dashboard ids by which to filter the
+        results.
+      isArray: true
+      name: dashboard_id
+      required: false
+      secret: false
+    - default: false
+      description: Panel ids by which to filter the results.
+      isArray: false
+      name: panel_id
+      required: false
+      secret: false
+    - default: false
+      description: Contained in the name of an alert.
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - default: false
+      description: 'A comma-separated list of states by which to filter the results.
+        The options are: all, no_data, paused, alerting, ok, pending.'
+      isArray: true
+      name: state
+      required: false
+      secret: false
+    - default: false
+      description: Limit response to X number of alerts.
```

Comment:
please look at other limit descriptions from other integrations and copy from them